libs/template/src/index.ts

```diff
@@ -3,7 +3,7 @@ export {
   priorityTickMap,
   SchedulingPriority,
 } from './lib/render-strategies';
-export { getZoneUnPatchedApi, isNgZone, isViewEngineIvy } from './lib/core';
```

Comment:
We need to add it somewhere to make it accessible by `apps/experiments`. Or remove this check from `config-pannel.component.ts`. Otherwise, experiments build will fail. And experiments build is running for each PR.